sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -313,7 +313,7 @@ class Analyzer(override val catalogManager: CatalogManager) extends RuleExecutor
       TimeWindowing ::
       SessionWindowing ::
       ResolveWindowTime ::
-      ResolveDefaultColumns(v1SessionCatalog) ::
+      ResolveDefaultColumns(v1SessionCatalog, catalogManager) ::
```

Comment:
`catalogManager` contains a `v1SessionCatalog` as well. Shall we expose it from `CatalogManager` instead?

Reply:
Sounds good, done!